examples/animatediff/tools/motion_module_convert.py

```diff
@@ -81,7 +81,11 @@ def _param_info_parser(param_info_txt):
     elif args.version == "v2":
         torch_names_txt = "./torch_mmv1_params.txt"
         ms_names_txt = "./ms_mmv1_params.txt"
+    elif args.version == "v3":
+        torch_names_txt = "./torch_mmv1_params.txt"
+        ms_names_txt = "./ms_mmv1_params.txt"
```

Comment:
Is it mmv3? And there is no new params txt added in this PR.

Reply:
Motion module `v3` has the same parameters names as `v1` and `v2`, so no need to add new params txt. I keep the version `v3` as name differentiation.